content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Use o redirecionamento de porta para acessar aplicativos em um cluster.
+content_type: task
+weight: 40
+min-kubernetes-server-version: v1.10
+---
+
+<!-- overview -->
+
+Esta página mostra como usar o `kubectl port-forward` para se conectar a um servidor MongoDB em execução em um cluster Kubernetes. Esse tipo de conexão pode ser útil para depuração de bancos de dados.
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* Instale o [MongoDB Shell](https://www.mongodb.com/try/download/shell).
+
+<!-- steps -->
+
+## Criando a implantação (*deployment*) e o serviço (service) do MongoDB
```

Comment:
I don´t think we need to have both 'deployment' and 'service' referenced when there is the translation. I asked this in our pt slack channel.